.github/workflows/ci.yml

```diff
@@ -0,0 +1,24 @@
+name: core-sdk CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        # add versions to array as we move support forward
+        node-version: [12.x]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm ci
```

Comment:
`npm ci` seemed to kick off `build` automatically locally. I included both anyway since the [starter workflow has it this way](https://github.com/actions/starter-workflows/blob/be562a72393b0ddb90e69cd3cd6b3ff79aaa8315/ci/node.js.yml).

Reply:
Seems fine to me. The "lint" and "test" commands also compile if necessary but the "build" does a "clean" first which doesn't seems like a bad idea.